### ISSUE TITLE
Adding cli param to disable usage tracking

### ISSUE THREAD

**Issue by lesaff** [burgerbuds/swiff#39]
I just noticed that `swiff` tracks usage. Is there a way to disable this? I understand the need of tracking but not all users would appreciate being tracked.

Thx

**Comment by ben-rogerson** [comment]
agreed, I'd like to remove it altogether - it was only added to count the amount of task calls so I could order them appropriately in the UI. There's no need for it now so next release it's highly likely to be stripped out.
